fontforgeexe/uiutil.c

```diff
@@ -528,27 +535,31 @@ return( GGadgetDispatchEvent(errdata.vsb,event));
       case et_expose:
```

Comment:
leaving this stuff in the switch for future compatibility? (At present `if ( event->type == et_expose ) {` would be simpler.)

Reply:
Oh I didn't even spot that when working on this. I think I'll leave it anyway, given it's the usual pattern for working with events.